.gitlab/pipeline/test.yml

```diff
@@ -492,3 +492,16 @@ test-syscalls:
       printf "The x86_64 syscalls used by the worker binaries have changed. Please review if this is expected and update polkadot/scripts/list-syscalls/*-worker-syscalls as needed.\n";
       fi
   allow_failure: false # this rarely triggers in practice
+
+subsystem-regression-tests:
```

Comment:
The pod needs to be scheduled to run with guaranteed CPU resources, otherwise the tests will be very unstable when the CI pipelines are loaded.

Reply:
The job has tag `benchmarks` which means that it runs every time on reference hw (`n2-standard-8`)